res/controllers/American Audio VMS4.midi.xml

```diff
@@ -270,13 +271,13 @@
          </control>
          <control><!-- on -->
             <group>[Channel1]</group>
-            <key>cue_play</key>
+            <key>cue_gotoandplay</key>
```

Comment:
This is redundant with holding down the cue button then pressing play. I think it would be more useful to use `start_stop` here.